sample-plugin/javascript/src/__tests__/e2e.spec.js

```diff
@@ -0,0 +1,55 @@
+require("regenerator-runtime")
+
+const { Web3ApiClient } = require("@web3api/client-js");
+const { jsExamplePlugin } = require("../../build");
+
+describe("e2e", () => {
+
+  let client;
+  const uri = "ens/sampleplugin.eth";
+
+  beforeAll(() => {
+    // Add the samplePlugin to the Web3ApiClient
+    client = new Web3ApiClient({
+      plugins: [
+        {
+          uri,
+          plugin: jsExamplePlugin({ defaultValue: "foo bar" })
+        }
+      ]
+    });
+  });
+
+  it("sampleQuery", async () => {
+    const result = await client.query({
```

Comment:
Let's use the `.invoke` syntax everywhere from now on: https://forum.polywrap.io/t/graphql-is-it-hindering-developer-experience/130